### ISSUE TITLE
Token time to live not working

### ISSUE THREAD

**Issue by Motard** [tymondesigns/jwt-auth#996]
I am using laravel, and I set in config/jwt.php ttl => 1, but I am not getting token expired after 1 minute I created the token. I do login, get a new token, then start making requests, supplying the received token, but all works, I mean, the requests are satisfied. I think that after 1 minute I should get a token expired response.

**Comment by sleenen** [comment]
Depends on what you do with the token; you using middleware to protect the routes?

**Comment by Motard** [comment]
Yes, I am using middleware. And I have tested that if I increase the TTL to anything bigger that one hour, that TTL is respected. But any TTL under an hour, is defaulting to one hour.

**Comment by sleenen** [comment]
Did you add logging to the handle function? To see if it actually goes in there and passes the code you pasted

**Comment by cqdplus** [comment]
let's set in .env file

JWT_SECRET=xxx
JWT_TTL=(time)
